.swiftlint.yml

```diff
@@ -0,0 +1,48 @@
+disabled_rules: # rule identifiers to exclude from running
```

Comment:
Not questioning the config, but curious where you sourced this.

Reply:
https://github.com/realm/SwiftLint#configuration

This seems to be the recommended configuration because without these it threw over 300 linting errors even though all I did was added two libraries and create a test.